teuthology/suite/placeholder.py

```diff
@@ -98,6 +98,9 @@ def _substitute(input_dict, values_dict):
             'sha1': Placeholder('suite_hash'),
         }
     },
+    'syslog': {
+        'ignorelist': ['WARNING*.*check_session_state', 'WARNING*.*__ceph_remove_cap'],
```

Comment:
These should be in `qa/cephfs/begin.yaml`. Please open a PR.

Reply:
I think we can just have this array empty.

Reply:
Done